src/draw/nxp/g2d/lv_draw_g2d.c

```diff
@@ -0,0 +1,317 @@
+/**
+ * @file lv_draw_g2d.c
+ *
+ */
+
+/**
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include "lv_draw_g2d.h"
+
+#if LV_USE_DRAW_G2D
+#include "../../../misc/lv_area_private.h"
+#include "g2d.h"
+#include "lv_g2d_buf_map.h"
+
+/*********************
+ *      DEFINES
+ *********************/
+
+#define DRAW_UNIT_ID_G2D 4
```

Comment:
Already in use by Dave2D.

```suggestion
#define DRAW_UNIT_ID_G2D 8
```